app/assets/stylesheets/metrics.scss

```diff
@@ -15,6 +15,16 @@ $autolab-subtle-red: #C87B7E;
   background-color: $autolab-subtle-red !important;
 }
 
+.icon.button {
+  color: #FFFFFF !important;
+  background-color: #767676 !important;
```

Comment:
Shall we make the button darker so that it stands out more when enabled and allows for a greater contrast when users hover over it?